app/views/application/_settings_menu.html.erb

```diff
@@ -14,5 +14,8 @@
     <li class="nav-item">
       <%= link_to t(".oauth2_authorizations"), oauth_authorized_applications_path, :class => "nav-link #{'active' if controller_name == 'oauth2_authorized_applications'}" %>
     </li>
+    <li class="nav-item">
+      <%= link_to t(".muted_users"), user_mutes_path, :class => "nav-link #{'active' if controller_name == 'user_mutes'}" %>
```

Comment:
We hide the "Muted messages" tab when there's no muted messages. Should we also hide the "Muted users" tab when you haven't muted anyone yet? There's not much value for this tab until you do (and the muting happens elsewhere), and most people are (hopefully) never going to need it.

Reply:
yup. I changed `messages/outbox.html.erb` to also reuse `messages/_messages_table.html.erb` and therefore pass a custom `inner_partial` and a new configuration – `columns` to `messages/_messages_table.html.erb`.

I'd understand if the view logic seems a bit overcomplicated now, but I think its still not too hard to read and therefore the message table logic is kept together in a small set of files.

See c0cbf1c28da68739bc854eb8249f5298ce16e8a5